Addon.Episerver.EnvironmentSynchronizer/Synchronizers/SiteDefinitions/SiteDefinitionSynchronizer.cs

```diff
@@ -110,5 +125,52 @@ private SiteDefinition GetExistingSiteDefinition(IEnumerable<SiteDefinition> exi
 
             return siteDefinition;
         }
+
+        private void SetForceLogin(SiteDefinition site, EnvironmentSynchronizerSiteDefinition siteDefinitionToUpdate)
+		{
+			var siteStartPageContentLink = site.StartPage;
+			if (siteStartPageContentLink != null)
+			{
+				IContentSecurityDescriptor securityDescriptor = (IContentSecurityDescriptor)_contentSecurityRepository.Get(siteStartPageContentLink).CreateWritableClone();
+
+                if (securityDescriptor != null)
+                {
+					if (securityDescriptor.IsInherited)
+					{
+						securityDescriptor.IsInherited = false;
+					}
+
+					var foundEveryoneRead = false;
+					var existingEntries = new List<AccessControlEntry>();
+					foreach (var entry in securityDescriptor.Entries)
+					{
+						Logger.Information($"Found AccessControlEntry {entry.Name}-{entry.Access} for site {siteDefinitionToUpdate.Name}.");
+						if (entry.Name == "Everyone")
+						{
+							foundEveryoneRead = true;
```

Comment:
How do you know that this security descriptor includes read access? If that's true, rename to remove "read" from variable name.